packages/data-point/lib/entity-types/entity-request/resolve.test.js

```diff
@@ -391,4 +391,21 @@ describe('resolve', () => {
       })
     })
   })
+
+  test('it should omit options.auth when encountering an error', () => {
```

Comment:
this test logs an error object that we could intercept if we mock `console.info` here (that's what we do in a few other tests in this file):

![screen shot 2018-01-29 at 7 03 50 pm](https://user-images.githubusercontent.com/3729628/35541122-3dc4aa10-0527-11e8-94a4-fc1f83615846.png)

Reply:
I agree with you on that, and I actually removed that `console.info` in https://github.com/ViacomInc/data-point/pull/186. on that branch, nothing is logged by default, but there's an [options.debug](https://github.com/raingerber/data-point/blob/304fca7a412900368e1295d9de9b44a62bb06ab3/packages/data-point/README.md#debugging) setting that can be set to `true` if you do want logging

Reply:
Maybe we should wrap all these in that setting then. 🤔